plantuml/src/browser/plantuml-grammar-contribution.ts

```diff
@@ -13,6 +13,7 @@ export const PLANTUML_LANGUAGE_NAME = 'PlantUML';
 
 @injectable()
 export class PlantumlGrammarContribution implements LanguageGrammarDefinitionContribution {
+    readonly scopeName = 'source.wsd';
```

Comment:
`source.plantuml` would be better to void another potential collision.

Reply:
nvm, I read your comment.